docs/topics/mpp/migrating-multiplatform-project-to-14.md

```diff
@@ -94,18 +94,14 @@ In future versions, the hierarchical project structure will become default for K
 
 ## For library authors
 
-### Check uploading to Bintray
+### Migrate from Gradle Bintray plugin to Maven Publish plugin
 
-The Bintray plugin doesn’t support publishing Gradle module metadata, but there are a couple of ways to get around this issue:
-
-* Migrate to `maven-publish` instead of `gradle-bintray-plugin` [as we did for kotlinx.serialization](https://github.com/Kotlin/kotlinx.serialization/commit/c5f1af6ad78a77fe5861588d9fb00b7d3a9bc3e5#diff-439aadfed1f3c340acdcc871c00258aeL5).
+If you're using `gradle-bintray-plugin` for library publication, migrate your projects to `maven-publish` plugin instead.
+See how we've done this for `kotlinx.serialization` [here](https://github.com/Kotlin/kotlinx.serialization/commit/c5f1af6ad78a77fe5861588d9fb00b7d3a9bc3e5#diff-439aadfed1f3c340acdcc871c00258aeL5).
```

Comment:
Please avoid using non-representative words such as `here` in links. They are bad for SEO. They are not indexed.

Reply:
Fixed the link text, thanks.